context/app/static/js/components/Detail/DetailLayout.jsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import styled from 'styled-components';
+import NoticeAlert from '../NoticeAlert';
+import TableOfContents from './TableOfContents';
+
+const FlexColumn = styled.div`
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  max-width: 1280px;
+`;
+
+const FlexRow = styled.div`
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+`;
+
+function DetailLayout(props) {
+  const {
+    displayViz,
+    displayProtocol,
+    displayMetadata,
+    displayMetadataTable,
+    displayFiles,
+    displayDag,
+    flashed_messages,
+    children,
+  } = props;
+
+  const updateSections = () => {
```

Comment:
From the outside world, it's not really an update... may just
```suggestion
  const getSections = () => {
```